x-pack/plugin/runtime-fields/src/main/java/org/elasticsearch/xpack/runtimefields/query/StringScriptFieldExistsQuery.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.runtimefields.query;
+
+import org.elasticsearch.xpack.runtimefields.StringScriptFieldScript;
+
+import java.util.List;
+
+public class StringScriptFieldExistsQuery extends AbstractStringScriptFieldQuery {
+    public StringScriptFieldExistsQuery(StringScriptFieldScript.LeafFactory leafFactory, String fieldName) {
+        super(leafFactory, fieldName);
+    }
+
+    @Override
+    public boolean matches(List<String> values) {
+        return false == values.isEmpty();
+    }
+
+    @Override
+    public String bareToString() {
+        return "*";
```

Comment:
I get it, but I find this cryptic. For instance looking at what DocValuesFieldExistsQuery does , that is clearer to me.

Reply:
I can keep the original `toString` impls.